packages/discovery-react-components/src/components/CIDocument/components/DetailsPane/DetailsPane.tsx

```diff
@@ -20,7 +20,7 @@ const DetailsPane: FC<DetailsPaneProps> = ({
   onActiveLinkChange
 }) => {
   return (
-    <div className={base}>
+    <div className={base} data-testid="detailsPane">
```

Comment:
Let's keep convention of [kebab case](https://en.wikipedia.org/wiki/Letter_case#Special_case_styles): `details-pane`